test/e2e/puppeteer.js

```diff
@@ -6,8 +6,6 @@ const jimp = require( 'jimp' );
 const fs = require( 'fs' );
 
 const port = 1234;
-const pixelThreshold = 0.1; // threshold error in one pixel
-const maxFailedPixels = 0.05; // total failed pixels
```

Comment:
I not recommend completely remove threshold in pixelmatch. Current solution maybe not strict, because it accept %5 of total failed pixels. You could just set maxFailedPixels = 0.02 to get %2 of total failed pixels, and decrease pixelThreshold = 0.05 as a rule to detect a defect pixel. But with threshold you can ran this code without *swiftshader* mode on GPU.